doc/developers/index.rst

```diff
@@ -716,8 +716,11 @@ is not met, an exception of type ``ValueError`` should be raised.
 ``y`` might be ignored in the case of unsupervised learning. However, to
 make it possible to use the estimator as part of a pipeline that can
 mix both supervised and unsupervised transformers, even unsupervised
-estimators are kindly asked to accept a ``y=None`` keyword argument in
+estimators need to accept a ``y=None`` keyword argument in
```

Comment:
Indeed, this doc was already here. I guess I only looked at a glance at what I thought was an issue, but not really.